portallocator/portallocator_windows.go

```diff
@@ -1 +1,10 @@
 package portallocator
+
+const (
+	StartPortRange = 60000
+	EndPortRange   = 65000
```

Comment:
Looks like these don't have to be exported

Also; given that constants are assigned for these, I assume there's no configurable ephemeral port range on Windows (as there's on Linux)?

Reply:
Looks like there is a configurable range; should that be used? https://support.microsoft.com/en-gb/help/929851/the-default-dynamic-port-range-for-tcp-ip-has-changed-in-windows-vista